app/services/ingest_csv.rb

```diff
@@ -104,6 +117,10 @@ def has_parent_identifier?
     @has_parent_identifier ||= csv.first.headers.include?("parent_roda_identifier_compound")
   end
 
+  def has_transactions?
+    @has_transactions ||= csv.first.headers.include?("transaction_type")
```

Comment:
Yeah - I didn't want to have too many files to avoid having to do a full deploy, and since we're controlling the CSV contents, I figured this was fine :)